package.json

```diff
@@ -22,7 +22,8 @@
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject",
-    "deploy": "npm run build&&gh-pages -d build"
+    "deploy-dev": "npm run build && gh-pages -d build",
```

Comment:
Tested, but not yet working. Some issues have come up with deployment that we're trying to track down.